### PR TITLE
doc(instance): improve create image doc

### PR DESCRIPTION
**Pull request by QuentinBrosse**

### CODE REVIEW

File: api/instance/v1/instance_sdk.go [hunk 1]
```diff
@@ -1220,17 +1220,17 @@ type VolumeSummary struct {
 
 // VolumeTemplate volume template
 type VolumeTemplate struct {
-	// ID the volumes unique ID
+	// ID uUID of the volume
```
Comment (loicbourgois) [inline api/instance/v1/instance_sdk.go]: `uUID` -> `UUID`

File: api/instance/v1/instance_sdk.go [hunk 2]
```diff
@@ -2063,21 +2063,21 @@ func (s *API) GetImage(req *GetImageRequest, opts ...scw.RequestOption) (*GetIma
 
 type CreateImageRequest struct {
 	Zone scw.Zone `json:"-"`
-
+	// Name name of the image
```
Comment (kindermoumoute) [inline api/instance/v1/instance_sdk.go]: double name

File: api/instance/v1/instance_sdk.go [hunk 3]
```diff
@@ -1220,17 +1220,17 @@ type VolumeSummary struct {
 
 // VolumeTemplate volume template
 type VolumeTemplate struct {
-	// ID the volumes unique ID
+	// ID uUID of the volume
 	ID string `json:"id,omitempty"`
-	// Name the volumes name
+	// Name name of the volume
```
Comment (kindermoumoute) [inline api/instance/v1/instance_sdk.go]: double name

File: api/instance/v1/instance_sdk.go [hunk 4]
```diff
@@ -2063,21 +2063,21 @@ func (s *API) GetImage(req *GetImageRequest, opts ...scw.RequestOption) (*GetIma
 
 type CreateImageRequest struct {
 	Zone scw.Zone `json:"-"`
-
+	// Name name of the image
 	Name string `json:"name,omitempty"`
-
+	// RootVolume uUID of the snapshot
```
Comment (loicbourgois) [inline api/instance/v1/instance_sdk.go]: `uUID` -> `UUID`

File: api/instance/v1/instance_sdk.go (hunk 1 above)
Comment (QuentinBrosse) [reply api/instance/v1/instance_sdk.go]: I open an internal ticket to fix the documentation generation.

File: api/instance/v1/instance_sdk.go (hunk 3 above)
Comment (QuentinBrosse) [reply api/instance/v1/instance_sdk.go]: I open an internal ticket to fix the documentation generation.

File: api/instance/v1/instance_sdk.go (hunk 4 above)
Comment (QuentinBrosse) [reply api/instance/v1/instance_sdk.go]: I open an internal ticket to fix the documentation generation.

File: api/instance/v1/instance_sdk.go (hunk 2 above)
Comment (QuentinBrosse) [reply api/instance/v1/instance_sdk.go]: I open an internal ticket to fix the documentation generation.

**Pull request merged**